build/wd_test.bzl

```diff
@@ -145,6 +133,12 @@ echo Using Python Snapshot
 def _wd_test_impl(ctx):
     is_windows = ctx.target_platform_has_constraint(ctx.attr._platforms_os_windows[platform_common.ConstraintValueInfo])
 
+    if ctx.file.sidecar:
+        if ctx.attr.python_snapshot_test:
+            return print("sidecar and python_snapshot_test currently cannot be used together")
```

Comment:
Why do we disallow this? This such errors deserve their own TODO's

Reply:
Just because we don't currently have any need for it and the generated script for something like that would be hard to write. I'll add a comment that says roughly that.